src/UI/Art/SpriteArtist/PlayerSetupTeamArtist.java

```diff
@@ -52,22 +55,23 @@ public static void draw(Graphics g, MapInfo mapInfo, IController controller)
     int panelBuffer = 3;
     int panelHeight = TeamPanel.PANEL_HEIGHT+panelBuffer;
 
-    // If we are moving from one option to another, calculate the intermediate draw location.
-//    if( animHighlightedPlayer != highlightedPlayer )
-//    {
-//      double slide = SpriteUIUtils.calculateSlideAmount(animHighlightedPlayer, highlightedPlayer);
-//      animHighlightedPlayer += slide;
-//    }
-
     // Find where the zeroth player CO should be drawn.
     // Shift from the center location by the spacing times the number of the highlighted option.
-    int playerYCenter = (int)(playerZoneYCenter - (highlightedPlayer * panelHeight));
+    int target = (int)(playerZoneYCenter - (highlightedPlayer * panelHeight));
+    if( null == player0YCenter ) player0YCenter = new SlidingValue(target);
+    long callTime = System.currentTimeMillis();
+    if( callTime - lastCallTime > 64 )
```

Comment:
Is there any reason you couldn't just sub in the "Is `controller` equal to the controller I had before?" logic that a lot of the other artists have?

Reply:
I'm not finding examples of that right now, but I don't think it would work anyway. If you hit `back` then `down` then `enter`, the controller won't have changed, and you'll see the team panels snap into the new position anyway (assuming you increase 64 by an order of magnitude or so).

Reply:
Whoops, that is all false. Never mind. Yeah, that's doable.

Reply:
If you were hoping I would forget, I have not.
I'm definitely in favor of avoiding basing logic on some arbitrary number of milliseconds

Reply:
What? No. Why would I think you forgot? There's a whole comment chain right here to remind you.

Patience.